frontend/src/components/Issue/logic/assignee.ts

```diff
@@ -0,0 +1,62 @@
+import { computed } from "vue";
+import {
+  EnvironmentId,
+  IssueCreate,
+  IssueType,
+  Pipeline,
+  PipelineApprovalPolicyPayload,
+  Policy,
+} from "@/types";
+import { useIssueLogic } from ".";
+import { usePolicyByEnvironmentAndType } from "@/store";
+
+export const useAllowProjectOwnerToApprove = () => {
+  const { create, issue, activeStageOfPipeline } = useIssueLogic();
+
+  const activeEnvironmentId = computed((): EnvironmentId => {
+    if (create.value) {
+      // When creating an issue, activeEnvironmentId is the first stage's environmentId
```

Comment:
activeEnvironmentId should be the environmentId of the first stage that has a task.

Reply:
Is that possible a stage has no task?🤔

Reply:
That's not possible currently.